source/common/filter/ratelimit.cc

```diff
@@ -43,7 +43,7 @@ Network::FilterStatus Instance::onNewConnection() {
     config_->stats().active_.inc();
     config_->stats().total_.inc();
     calling_limit_ = true;
-    client_->limit(*this, config_->domain(), config_->descriptors(), EMPTY_STRING);
+    client_->limit(*this, config_->domain(), config_->descriptors(), {"", ""});
```

Comment:
{} should work, but either way, I should have commented on this before, define a static const empty context, and pass it here.